spec/javascript/controllers/feedback_controller_spec.js

```diff
@@ -1,5 +1,5 @@
 import { Application } from 'stimulus';
-import PageHelpfulController from 'page_helpful_controller';
+import PageHelpfulController from 'feedback_controller';
 
 describe('PageHelpfulController', () => {
```

Comment:
Missed a few references to `PageHelpfulController` here

Reply:
Good catch, I've updated the spec file.